composer.json

```diff
@@ -50,6 +50,7 @@
     },
     "config": {
         "sort-packages": true,
+        "bump-after-update": "dev",
```

Comment:
Do we need to constantly bump dependencies to exact version? What's the benefit?

Reply:
It's work for dev dependencies and gives more guarantees that package developers used same package versions in own environment.